libminifi/src/ThreadedSchedulingAgent.cpp

```diff
@@ -88,6 +88,7 @@ void ThreadedSchedulingAgent::schedule(std::shared_ptr<core::Processor> processo
 
   auto sessionFactory = std::make_shared<core::ProcessSessionFactory>(processContext);
 
+  processor->validateAnnotations();
```

Comment:
given our direction towards all kinds of flow-verification, this could be moved into either a `FlowController::verifyFlow(ProcessGroup&)` or `ProcessGroup::verify` (the former might be better as then we could access the services and repositories all at once) which could then be expanded as needed (calling it from `FlowController::load`)

Reply:
Done.